static/survey.yaml

```diff
@@ -0,0 +1,498 @@
+items:
```

Comment:
Right now it is ok to embed the file in bundle of files sent to the browser, but I don't think we would like to that long term.
It will be best to have this application being server side rendered and make this application read the `survey.yaml` from the backend and decide what kind of or subset of data we want to send to the frontend as a web page.